dtmi/azureiot/phone_as_a_device/phoneasadevice-1.json

```diff
@@ -0,0 +1,175 @@
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "@type": "Command",
+            "commandType": "asynchronous",
+            "displayName": {
+                "en": "Clear Connectivity"
+            },
+            "name": "clearConnectivity"
+        },
+        {
+            "@type": "Property",
+            "displayName": {
+                "en": "telemetry Interval (seconds)"
+            },
+            "name": "telemetryInterval",
+            "schema": {
+                "@type": "Enum",
+                "valueSchema": "integer",
+                "enumValues": [
+                    {
+                        "name": "five",
+                        "displayName": {
+                            "en": "5 Seconds (Default)"
+                        },
+                        "enumValue": 5000
+                    },
+                    {
+                        "name": "ten",
+                        "displayName": {
+                            "en": "10 Seconds"
+                        },
+                        "enumValue": 10000
+                    },
+                    {
+                        "name": "twenty",
+                        "displayName": {
+                            "en": "20 Seconds"
+                        },
+                        "enumValue": 20000
+                    },
+                    {
+                        "name": "thirty",
+                        "displayName": {
+                            "en": "30 Seconds"
+                        },
+                        "enumValue": 30000
+                    }
+                ]
+            },
+            "writable": true
+        },
+        {
+            "@type": "Property",
+            "displayName": {
+                "en": "Application Theme"
+            },
+            "name": "theme",
+            "schema": {
+                "@type": "Enum",
+                "valueSchema": "integer",
+                "enumValues": [
+                    {
+                        "name": "light",
+                        "displayName": {
+                            "en": "Light"
+                        },
+                        "enumValue": 1
+                    },
+                    {
+                        "name": "dark",
+                        "displayName": {
+                            "en": "Dark"
+                        },
+                        "enumValue": 2
+                    },
+                    {
+                        "name": "system",
+                        "displayName": {
+                            "en": "System-default"
+                        },
+                        "enumValue": 0
+                    }
+                ]
+            },
+            "writable": true
+        }
+    ],
+    "displayName": {
+        "en": "IoT Plug and Play mobile"
+    },
+    "@context": [
+        "dtmi:iotcentral:context;2",
```

Comment:
if we are not using the central context, I'd suggest to remove it.
nit: I prefer showing the context at the top